### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by championchap** [championchap/Bramble#8]
Hahahahaha

**Comment by erikwatson** [comment]
Migrated to TypeScript and while that is not documentation exactly, it does help specify exactly what our objects and functions look like.

**Comment by erikwatson** [comment]
I've started working on the Wiki.

The wiki will attempt to explain in plain English and with code examples how the library is intended to be used.

So far there is a page for getting started, one for reading user input and one for loading assets. 

More to come, particularly as the api settles down.
